src/FixedDataTableCellGroup.js

```diff
@@ -198,20 +219,20 @@ class FixedDataTableCellGroupImpl extends React.Component {
         className={cx('fixedDataTableCellGroupLayout/cellGroup')}
         style={style}
       >
-        {cells}
+        {this._staticCells}
       </div>
     );
   }
 
-  _renderCell = (
-    /*number*/ rowIndex,
-    /*number*/ height,
-    /*object*/ columnProps,
-    /*object*/ cellTemplate,
-    /*number*/ left,
-    /*string*/ key,
-    /*number*/ columnGroupWidth
-  ) /*object*/ => {
+  _renderCell = (/*number*/ key, /*number*/ columnIndex) /*object*/ => {
```

Comment:
I cleaned this method to just accept just the minimal parameters. 
All the other ones are either derivable based on these parameters or are already computed in `this.props`.